src/client.js

```diff
@@ -80,7 +80,7 @@ export async function connect(myVatID, addr, commandfile) {
       ',',
     )}`,
   );
-  const source = pullStream.values([
+  pullStream.values([
```

Comment:
make this a `_source`. It's used by the commented-out first entry of the `pullStream` call down below (the one whose second entry is `s2`). When enabled, it adds a few debug messages at the beginning of the transcript, to make sure its getting written to a file properly.. might be good to leave that around for now, just disabled.